src/ape/api/transactions.py

```diff
@@ -200,6 +203,56 @@ def decode_logs(self, abi: Union[EventABI, "ContractEvent"]) -> Iterator[Contrac
 
         yield from self.provider.network.ecosystem.decode_logs(abi, self.logs)
 
+    @property
+    def events(self):
```

Comment:
Should we call these `logs` instead of events? I am used to thinking of the event as the definition in the contract and the log as the data in python

Reply:
We already have `.logs` property, so this makes sense. It's basically decoding all the logs according to the event definitions in the contract cache

Reply:
I am guessing this should actually be in `ape_ethereum.transactions.Receipt`